projects/mercury/src/collections/CollectionInformationDrawer.js

```diff
@@ -278,51 +276,17 @@ type CollectionInformationDrawerProps = {
 };
 
 export const CollectionInformationDrawer = (props: CollectionInformationDrawerProps) => {
-    const {
-        collection, loading, atLeastSingleCollectionExists, setHasCollectionMetadataUpdates,
-        inCollectionsBrowser, path, showDeleted
-    } = props;
-
-    const paths = getPathHierarchy(path);
-
-    if (!collection) {
-        return atLeastSingleCollectionExists && inCollectionsBrowser
-            && <EmptyInformationDrawer message="Select a collection to display its metadata" />;
-    }
-
-    const hasEditRight = collection && collection.canWrite;
+    const {path: metadataPath, showDeleted} = props;
 
     return (
         <>
-            <CollectionDetails
-                collection={collection}
-                onChangeOwner={props.onChangeOwner}
-                loading={loading}
-                setBusy={props.setBusy}
+            <PathMetadata
```

Comment:
Does it show only single card now? 
Why not keep the `paths.map((metadataPath, index) => (`, just remove the collection details wrapper? 
It could be done like in ExternalStorageInformationDrawer

Reply:
What is the purpose of this, show all metadata of the complete path? And will this scale if the path is 9 steps deep?

Reply:
So far it worked fine, we can check if we can store the metadata that was already fetched in memory or add some improvements there. 
It was useful to see metadata of parent entities, but if this causes some performance issues, indeed we will have to remove it.